tools/sggolangcilint/golangci.yml

```
@@ -1,5 +1,7 @@
 run:
   timeout: 10m
+  build-tags:
+    - wireinject
```

```suggestion
  build-tags:
    - wireinject
    - tools
```

Perhaps add this one for `tools.go` too?

---

I don't think the tools file is necessary in this case. It's a lot more based on internal convention and different places might used it differently

---

Allright then, feel free to resolve this!